overlore/llm/open_ai.py

```diff
@@ -88,4 +94,20 @@ async def generate_townhall_discussion(
             else ""
         )
 
-        return (await self.request_prompt(systemPrompt, userPrompt), systemPrompt, userPrompt)
+        return (await self.request_prompt(systemPrompt, userPrompt, "gpt-4-1106-preview"), systemPrompt, userPrompt)
+
+    async def generate_npc_profile(self) -> str:
+        "trait_type == either 'positive' or 'negative'. Otherwise, GPT calls only give out positive traits"
+        trait_type = TRAIT_TYPE[random.randrange(2)]
+
+        formatted_sex_list = ", ".join(SEX)
```

Comment:
Are the different sexes likely to change? Otherwise let's just use male or female

Reply:
Probably not indeed, changed.